garglk/cheapglk/cgdate.cpp

```diff
@@ -32,10 +32,13 @@
 #include <string.h>
 #include <stdlib.h>
 #include <time.h>
-#include <sys/time.h>
 #include "glk.h"
 #include "garglk.h"
 
+#ifndef _WIN32
+#include <sys/time.h>
+#endif
```

Comment:
As below, I think this should be included for MinGW, and only removed with MSVC.